lint/witnesslinter.py

```diff
@@ -13,4 +13,4 @@
 from witnesslint import linter  # noqa: E402 raises, but prevents bytecode generation
 
 if __name__ == "__main__":
-    sys.exit(linter.main(sys.argv))
+    linter.main()
```

Comment:
`linter.main()` will call `sys.exit()` anyway, so there is no need to do it in `witnesslinter.py` as well. In fact, the latter is not part of the package, so this has to be done in `linter.py`.

Reply:
Yes, but this limits the reusability of the main function in `linter.py`. The - admittedly academic (since probably YAGNI here) - goal is to write the main function in such a way that someone else can also reuse that function in a different context.

>  so this has to be done in linter.py

I think this is not actually the case. The setup.py that poetry creates will mark this function as an "entry point", which will lead to the following script to be created once I install that package:
```
#!/home/god/virtualenv/bin/python
# -*- coding: utf-8 -*-
import re
import sys
from witnesslint.linter import main
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

So that one actually calls `sys.exit` with the return code of `main()`. This strongly suggests that rather than calling `sys.exit` inside `main()`, we should return the return code instead and also call `sys.exit` from witnesslinter.py in the same way. For reusability one would usually not hardcode any print statements, or at least make the `file` parameter of these configurable via a keyword parameter such that the person reusing the code can decide whether and where these lines should land up.